.golangci.yml

```diff
@@ -53,7 +53,6 @@ linters:
   enable:
     - bidichk
     - bodyclose
-    - depguard
```

Comment:
We already made this change to `atomic-swap`. With the latest release, you have to provide a config for depguard, because the default config tries to prevent dependencies on testify.